docs/docs/examples-tutorials/recipes/running-tracetest-with-azure-app-insights-collector.md

```diff
@@ -0,0 +1,237 @@
+# Running Tracetest with Azure App Insights (Node.js + OpenTelemetry Collector)
+
+:::note
+[Check out the source code on GitHub here.](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-azure-app-insights-collector)
+:::
+
+[Tracetest](https://tracetest.io/) is a testing tool based on [OpenTelemetry](https://opentelemetry.io/) that allows you to test your distributed application. It allows you to use data from distributed traces generated by OpenTelemetry to validate and assert if your application has the desired behavior defined by your test definitions.
+
+[Azure Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) is an extension of Azure Monitor and provides application performance monitoring (APM) features. APM tools are useful to monitor applications from development, through test, and into production in the following ways:
+
+- Proactively understand how an application is performing.
+- Reactively review application execution data to determine the cause of an incident.
+
+[OpenTelemetry Collector Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) - The official OpenTelemetry Distribution for packages outside of the core collector.
+
+## Sample Node.js App with Azure App Insights, The OpenTelemetry Collector and Tracetest
+
+This is a simple quick start guide on how to configure a Node.js app to use instrumentation with traces and Tracetest for enhancing your E2E and integration tests with trace-based testing. The infrastructure will use Azure App Insights as the trace data store, the OpenTelemetry Collector to process and route the telemetry data and a Node.js app to generate it.
+
+## Prerequisites
+
+You will need [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on your machine to run this quick start app!
+
+And the [App Insights Instrumentation Key](https://learn.microsoft.com/en-us/azure/bot-service/bot-service-resources-app-insights-keys?view=azure-bot-service-4.0) from your instance.
+
+## Project Structure
+
+The project is built with Docker Compose.
+
+### 1. Node.js App
+
+The `Dockerfile` in the root directory is for the Node.js app.
+
+### 2. Tracetest
+
+The `docker-compose.yaml` file, `tracetest.provision.yaml`, and `tracetest-config.yaml` in the `root` directory are for the setting up the Node.js App and Tracetest.
+
+### Docker Compose Network
+
+All `services` in the `docker-compose.yaml` are on the same network and will be reachable by hostname from within other services.
+
+## Node.js App
+
+The Node.js app is a simple Express app, contained in the `src/index.js` file.
+
+It is instrumented using the [Official OpenTelemetry Node.js SDK](https://www.npmjs.com/package/@opentelemetry/sdk-node) wrapping the application code to send telemetry data to the OpenTelemetry Collector.
+
+The key following is the instrumentation section from the `src/tracing.js` file.
```

Comment:
```suggestion
The following key is the instrumentation section from the `src/tracing.js` file.
```
Is 'key' the correct word or should it be 'code'?

Reply:
I'd say we should use "code".